src/main/java/core/basesyntax/service/operations/BalanceOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {

```

Comment:
It is recommended to add a JavaDoc comment explaining what this class does.

Reply:
ok, done